apphub/image_classification/cifar10_fast/cifar10_fast_tf.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""
+The FastEstimator implementation of cifar10-fast model
+ref: https://github.com/davidcpage/cifar10-fast
+"""
+import tempfile
+
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras.mixed_precision import experimental as mixed_precision
+from tensorflow.python.keras import backend, layers
+from tensorflow.python.keras.regularizers import l2
+
+import fastestimator as fe
+from fastestimator.dataset import NumpyDataset
+from fastestimator.dataset.data.cifar10 import load_data
+from fastestimator.op import NumpyOp
+from fastestimator.op.numpyop import CoarseDropout, HorizontalFlip, Minmax, Normalize, PadIfNeeded, RandomCrop, \
+    Sometimes
+from fastestimator.op.tensorop.loss import CrossEntropy
+from fastestimator.op.tensorop.model import ModelOp, UpdateOp
+from fastestimator.pipeline import Pipeline
+from fastestimator.trace import Trace
+from fastestimator.trace.io import BestModelSaver
+from fastestimator.trace.metric import Accuracy
+
+policy = mixed_precision.Policy('mixed_float16')
+mixed_precision.set_policy(policy)
+
+
+class SmootOneHot(NumpyOp):
```

Comment:
"SmoothOneHot", also, this should go into our op folder

Reply:
rather, a parameterized version of this should go in our op folder

Reply:
done